src/pages/home/report/ReportActionCompose.js

```diff
@@ -56,7 +56,6 @@ class ReportActionCompose extends React.Component {
         this.updateComment = this.updateComment.bind(this);
         this.debouncedSaveReportComment = _.debounce(this.debouncedSaveReportComment.bind(this), 1000, false);
         this.debouncedBroadcastUserIsTyping = _.debounce(this.debouncedBroadcastUserIsTyping.bind(this), 100, true);
-        this.submitForm = this.submitForm.bind(this);
```

Comment:
this was just a duplicate binding that I can across when looking into the issue

Reply:
Nice catch! Funny that slipped by a previous review ;)